sensor_msgs/Image image
---
bool success
string status_message